recipes/andes/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "andes" %}
+{% set version = "0.6.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 45b384a1c3c4dfc9f5bd05a736278c3cce678b82316caf771c3aeb97d98da807
+
+build:
+  number: 0
+  entry_points:
+    - andes = andes.main:main
+    - andesplot = andes.plot:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py<35]
+
+requirements:
+  host:
+    - cvxopt
+    - matplotlib
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy
```

Comment:
This probably just needs to be python and pip since this looks like its a pure python package